.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+name: Eslint Check
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
Wouldn't it make sense to run this action on all pull requests including this one?

Afaict, this will only run on `master` hence _after_ something was merged

Reply:
good point, not sure why I only had it for master branch.

Running on every pull request now, I think a maintainer needs to manually run it for this PR since it isn't merged in yet